R/module_rds.R

```diff
@@ -0,0 +1,118 @@
+#' rds tab UI
+#' Module to create a tabset panel to allow the download of the rds file
+#' @param id ID for shiny module namespacing
+#' @noRd
+rdsUI = function(id) {
+  ns = shiny::NS(id)
+  # Plots tab panel
+  shiny::tabPanel("RDS Files",
+                  # drop down menu to select plot
+                  shiny::br(),
+                  shiny::uiOutput(ns("choose_rds")),
+                  # display plot
+                  shiny::wellPanel(
+                    shiny::fluidRow(shiny::column(
+                      12,
+                      align = "center",
+                      shiny::uiOutput(ns("display_rds")),
+                      style = "height:400px;")),
+                    style = "background: white"
+                  ),
+                  # download button to download current plot
+                  shiny::br(),
+                  shiny::fluidRow(
+                    shiny::column(12,
+                                  align = "center",
+                                  shiny::uiOutput(ns("download_rds_button")) #nolint
+                    )
+                  )
+  )
+}
+
```

Comment:
The UI components are extremely similar between this module and the plots/tables modules. Could you add a function to define the wellPanels:

```r
display_panel = function(...) {
  shiny::wellPanel(
      shiny::fluidRow(
        shiny::column(
           12,
           align = "center",
           ...,
           style = "height:400px;"
        )
      ),
      style = "background: white"
   )
}
```
Then replace the different uses of this across the three modules.

An alterrnative would be to introduce a class on the 'column', and modify some CSS to set the align/style.

Reply:
(Feel free to change the function name)

Reply:
I've added in this function and switched the code in the modules.